recipe/meta.yaml

```diff
@@ -33,7 +33,10 @@ outputs:
       script: cd "${SRC_DIR}/pennylane" && {{ PYTHON }} -m pip install . --no-deps -vv
     requirements:
       build:
-        - python
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
```

Comment:
@conda-forge/pennylane The compilers were missing here, so this would explain why the PRs were sensitive to [the change of the CI runner image](https://github.com/conda-forge/pennylane-feedstock/pull/17#issuecomment-1495021524)... I think it's clear that it was a bug in the recipe that we missed until now (exposed by the need of cross-compiling for M1/M2).

Reply:
I take it back. Misread the output name... My bad 🤯